src/com/kainos/buildasub/oop/Sandwich.java

```diff
@@ -0,0 +1,65 @@
+package com.kainos.buildasub.oop;
+
+import java.util.List;
+
+public class Sandwich {
+
+    private String Bread = "";
```

Comment:
Default values aren't necessary for the sub, I don't think you can go through and buy a sub without bread and meat. It might be worth adding a constructor instead